src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static add(creature) {
+    this.alive.push(creature);
+  }
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.add(this);
```

Comment:
I suggest to push 'this' directly to 'Animal.alive' array instead of creating  static method